.github/workflows/nix-dev-cache.yaml

```diff
@@ -24,6 +24,11 @@ jobs:
           - macOS-14
     steps:
     - uses: actions/checkout@v4
+    - name: set up nix mount points on Linux
+      if: runner.os == 'Linux'
```

Comment:
Why limit this to Linux?
- Because macOS `mount` doesn’t support `--bind`?
- Because we don’t need it on macOS? (I try to do things as similarly across platforms as possible, though)
- Because there’s no `/mnt` partition on macOS runners?

Could there be a comment about whatever it is?

Reply:
Yeah it's both \#2 and \#3; not sure about #1 but skimming `man mount` on mac, it also seems to be true.